### ISSUE TITLE
img.dev.storj.io doesn't allow to push new images

### ISSUE THREAD

**Issue by elek** [storj/distribution#1]
```
docker push img.dev.storj.io/dev/checkerng
Using default tag: latest
The push refers to repository [img.dev.storj.io/dev/checkerng]
74c4981f3a0b: Layer already exists 
3b0573825467: Layer already exists 
b530df93bb93: Layer already exists 
dc049a46d1b4: Layer already exists 
b9672b062964: Layer already exists 
fa39d522708b: Layer already exists 
822ee476c182: Layer already exists 
4ff4865fda68: Layer already exists 
c9552617ed8e: Layer already exists 
f26de0c16474: Layer already exists 
a502b4e0c279: Layer already exists 
ed834f4d5944: Layer already exists 
30768046a6f2: Layer already exists 
f4554cc614c8: Layer already exists 
051c408be63b: Layer already exists 
8d70467291d4: Layer already exists 
d925e0fae4e6: Layer already exists 
received unexpected HTTP status: 500 Internal Server Error
```

**Comment by elek** [comment]
Error:

```
time="2023-05-03T10:02:12.044135981Z" level=error msg="response completed with error" auth.user.name= err.code=unknown err.detail="storj: uplink: object not found: metabase: object not found: metabase: sql: no rows in result set" err.message="unknown error" go.version=go1.17.12 http.request.host=img.dev.storj.io http.request.id=aa3006ba-66e7-45ca-9a26-52d944f68418 http.request.method=HEAD http.request.remoteaddr=10.118.0.3 http.request.uri=/v2/dev/gateway-mt/manifests/latest http.request.useragent=containerd/1.6.12 http.response.contenttype=application/json http.response.duration=205.861884ms http.response.status=500 http.response.written=104 instance.id=a9bc4eaf-5f07-498f-8a5e-3e01576dc7b4 vars.name=dev/gateway-mt vars.reference=latest version=2.7.0-SNAPSHOT-ca40c527
```

Jeff already identified similar problem with a unit test:


> it seems like there's an extra "metabase" wrapper happening and so our string based error checking is now failing 

> oh yep, this did it https://review.dev.storj.io/c/storj/storj/+/10244/6/satellite/metabase/commit.go

**Comment by elek** [comment]
Fixed with deployment of v1.78.2 which contains the [fix](https://github.com/storj/storj/commit/2690efdcc2943d5f73c20c04aeb0ab62dc5732b5)
